configs/alephium/mainnet.json

```diff
@@ -1,7 +1,7 @@
 {
   "chainId": 255,
   "groupIndex": 0,
-  "initRewards": 0,
+  "initRewards": 1000,
```

Comment:
Do we have a TxScript to topup ALPH rewards?

Reply:
Yes, the script is here: https://github.com/alephium/wormhole-fork/blob/master/alephium/contracts/token_bridge_scripts/add_rewards.ral